datasources/National_Datasets/Social_and_Economic.ejs

```diff
@@ -100,26 +100,46 @@
                     }
                   }
                 },
-                 {
-                  "name": "Taxation Statistics 2015-2016",
-                  "id": "Root Group/National Data Sets/Social and Economic/Taxation Statistics 2015-2016",
-                  "type": "ckan-resource",
-                  "url": "https://data.gov.au",
-                  "resourceId": "859d8053-7f62-4bd7-82ed-8c1cd9156a2b",
-                  "itemProperties": {
-                    "showWarnings": false,
-                    "tableStyle": {
-                      "dataVariable": "Average taxable income or loss",
-                      "timeColumn": null,
-                      "colorPalette": "5-class PuBu",
-                      "columns": {
-                        "Postcode": {
-                          "name": "postcode_2011"
-                        }
+                {
+                "name": "Taxation Statistics 2015-2016",
+                "id": "Root Group/National Data Sets/Social and Economic/Taxation Statistics 2015-2016",
+                "type": "ckan-resource",
+                "url": "https://data.gov.au",
+                "resourceId": "859d8053-7f62-4bd7-82ed-8c1cd9156a2b",
+                "itemProperties": {
+                  "showWarnings": false,
+                  "tableStyle": {
+                    "dataVariable": "Average taxable income or loss",
+                    "timeColumn": null,
+                    "colorPalette": "5-class PuBu",
+                    "columns": {
+                      "Postcode": {
+                        "name": "postcode_2011"
                       }
                     }
                   }
                 }
+              },
+              {
+              "name": "Taxation Statistics 2016-2017",
+              "id": "Root Group/National Data Sets/Social and Economic/Taxation Statistics 2015-2016",
```

Comment:
I think the `id` should be "... 2016-2017". When there are items with duplicate ids, the catalog will only show the last one.

Reply:
I knew I asked someone to review this for good reason :) well spotted, fixed now